batchflow/research/domain.py

```diff
@@ -254,7 +254,7 @@ def __init__(self, domain=None, **kwargs):
         self.n_updates = 0
         self.additional = True
         self.create_id_prefix = False
-        self.random_state = None
+        self.rng = None
```

Comment:
There is a chain of `random_state` variables and their SeedSequences in `batchflow`:
```
---Pipeline
---Dataset
------Batch
---------inbatch_parallel workers (threads / processes / for-items)
```

And `Research` is even one level above that.

Have you read this entire chain of properties before changing it?